heron/statefulstorages/tests/java/com/twitter/heron/statefulstorage/localfs/LocalFSStorageTest.java

```diff
@@ -0,0 +1,153 @@
+// Copyright 2016 Twitter. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.twitter.heron.statefulstorage.localfs;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import com.google.protobuf.ByteString;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+
+import com.twitter.heron.common.basics.FileUtils;
+import com.twitter.heron.proto.ckptmgr.CheckpointManager.GetInstanceStateRequest;
+import com.twitter.heron.proto.ckptmgr.CheckpointManager.InstanceStateCheckpoint;
+import com.twitter.heron.proto.ckptmgr.CheckpointManager.SaveInstanceStateRequest;
+import com.twitter.heron.proto.system.PhysicalPlans;
+import com.twitter.heron.spi.statefulstorage.Checkpoint;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Matchers.any;
+import static org.mockito.Matchers.anyBoolean;
+import static org.mockito.Matchers.anyString;
+import static org.mockito.Matchers.eq;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.spy;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.when;
+
+@RunWith(PowerMockRunner.class)
+@PrepareForTest(FileUtils.class)
+public class LocalFSStorageTest {
```

Comment:
The `LocalFSStorageTest` here is in consistency with the class it's testing,i.e. `LocalFSStorage`.

If we make this change, then we also need to update the `LocalFSStorage.java` to `LocalFileSystemStorage.java`.

Let me know your thoughts.